js/stores/user_store.js

```diff
@@ -0,0 +1,60 @@
+var EventEmitter = require('events').EventEmitter;
+var assign = require('object-assign');
+var AppDispatcher = require('../dispatcher/app_dispatcher');
+var ActionTypes = require('../constants/constants').ActionTypes;
+
+var _users = [
+  {
+    id: 1,
+    name: 'rory'
+  },
+  {
+    id: 2,
+    name: 'iz'
+  },
+  {
+    id: 3,
+    name: 'jack'
+  }
+];
+
+var _localUser = {
+  id: 3,
+  name: 'jack'
+};
+
+var UserStore = assign({}, EventEmitter.prototype, {
+  localUser: function() {
+    return _localUser;
+  },
+
+  all: function() {
+    return _users;
+  },
+
+  find: function(id) {
+    for (i in _users) {
+      if (_users[i].id === id) {
+        return _users[i];
+      }
+    }
+  },
+
+  renameLocalUser: function(newName) {
+    var oldUser = _localUser;
+    _localUser = {
+      id: oldUser.id,
+      name: newName
+    };
+    this.find(oldUser.id).name = newName
```

Comment:
`find` is ES6.. to be safe, we could go with the ol' `filter(blah)[0]`.. but i was going to put in Babel support anyway so that we can use ES6, so we can just wait on that if you prefer